src/managed/CommonManaged.props

```diff
@@ -11,6 +11,8 @@
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
     <IncludeSymbols>true</IncludeSymbols>
     <Serviceable>true</Serviceable>
+    <PackageLicenseUrl>https://github.com/dotnet/core-setup/blob/master/LICENSE.TXT</PackageLicenseUrl>
```

Comment:
Microsoft.NETCore.App's license url is:

`<licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>`.

We should probably use the same fwlink.